config/exporters/monitoring/grafana/base/prometheus-exporter-service.yaml

```diff
@@ -7,7 +9,7 @@ spec: {}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: exporter-sa
+  name: kube-rbac-proxy
```

Comment:
What's the reason for renaming? As far as I understand, this is the service account that allows us to access the grafana datasource. I would not expect it to change in this task. I'd only expect that service account that allows prometheus to read from the exporter to be affected here.

Reply:
I tried reverting to `exporter-sa` and its throwing gin a 401 forbidden error.

Reply:
In previous PRs, we changed the code so that the exporter is reading data from grafana. A service account was required for that. I believe this is that service account.
Now we want to make the exporter itself secured and for that we need another service account.

Those two service accounts should be unrelated. We should not change the first one to make the second one work. Not sure what is throwing the error now, but it should not be related.

Reply:
@yftacherzog 
I added a new service account & I tried changing the serviceaccount within the Deployment to `exporter-sa` and the Target is  down and not being scraped for metrics.

Next, I left the `serviceAccountName` within the `Deployment` to remain as `kube-rbac-proxy`, that way the two service accounts `exporter-sa` & `kube-rbac-proxy` (kube-rbac-proxy for reading data from Grafana) remains unrelated. After deploying it in dev, the target is currently up and being actively scraped for metrics.